src/Listener/SaveSuspensionToDatabase.php

```diff
@@ -20,6 +21,21 @@ class SaveSuspensionToDatabase
     use AssertPermissionTrait;
 
     /**
+     * @var SuspendValidator
+     */
+    protected $validator;
+
+    /**
+     * SaveSuspensionToDatabase constructor.
```

Comment:
> Smart IDE's will assist you with auto completion if you have them, if not there's no way of knowing what class it is unless you fully backtrace into the code.

I don't understand? Just to be clear, we are referring specifically to the "SaveSuspensionToDatabase constructor." line - that's unnecessary and serves no purpose. So instead of

```php
    /**
     * SaveSuspensionToDatabase constructor.
     *
     * @param SuspendValidator $validator
     */
    public function __construct(SuspendValidator $validator)
    {
        $this->validator = $validator;
    }
```

it should just be:

```php
    /**
     * @param SuspendValidator $validator
     */
    public function __construct(SuspendValidator $validator)
    {
        $this->validator = $validator;
    }
```